rapid_pro_tools/rapid_pro_client.py

```diff
@@ -83,9 +79,12 @@ def get_archive(self, archive_metadata):
                     serialized_run["start"] = None
                     
                 results.append(Run.deserialize(serialized_run))
-        else:
-            # TODO: Support deserializing messages
-            assert False, "Deserializing messages archives is not implemented yet"
+        
+        if archive_metadata.archive_type == "message":
```

Comment:
Can you change this to:
```python
else:
    assert archive_metadata.archive_type == "message", "Unsupported archive type, must be either 'run' or 'message'"
```
That way we crash if we get an unknown archive type, rather than silently returning an empty list.